ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/cucushift-installer-rehearse-aws-ipi-workflow.yaml

```diff
@@ -2,12 +2,8 @@ workflow:
   as: cucushift-installer-rehearse-aws-ipi
   steps: 
     pre:
-    - chain: ipi-conf-aws
-    - chain: ipi-install
+    - chain: openshift-qe-ci-profiles-aws-ipi-provision
```

Comment:
I prefer to use the same dir as where workflow yaml is being saved, e.g: `cucushift/installer/rehearse/aws/ipi/provision` and `cucushift/installer/rehearse/aws/ipi/deprovision`, so that user can easily find everything in one parent directory.

Reply:
OK, the chain under `provision` and `deprovision` will be the _special_ chain, will update